xbmc/video/VideoInfoTag.h

```diff
@@ -291,6 +302,7 @@ class CVideoInfoTag : public IArchivable, public ISerializable, public ISortable
 
   int m_playCount;
   CBookmark m_resumePoint;
+  static const int VALUE_NOT_SET = -10000;
```

Comment:
But the name should be changed to be clear as @Montellese suggested. Something like `PLAY_COUNT_NOT_SET`